app/services/api/v3/ecf/participants_query.rb

```diff
@@ -38,7 +38,7 @@ def participants_from(paginated_join)
                   .joins(left_outer_join_participant_profiles)
                   .joins(left_outer_join_induction_records)
                   .joins("JOIN (#{latest_induction_records_join.to_sql}) AS latest_induction_records ON latest_induction_records.latest_id = induction_records.id")
-                  .joins("INNER JOIN (#{paginated_join.to_sql}) as tmp on tmp.id = users.id")
+                  .where(users: { id: paginated_join.map(&:id) })
```

Comment:
will this become an issue if we map over 3000 ids?

Reply:
I re-ran the test with 3000 IDs and it held up; I benchmarked the averages locally against snapshot (in development so no caching etc) and it gave like for like performance with the non-countless paginate operation (I'm hoping in prod it'll be a bit faster)